test-crates/duchess-java-tests/.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[env]
+CLASSPATH = { value = "target/java", relative = true, force = true }
```

Comment:
Could you check if this file is really required? Removing it doesn't seem to cause issues when I tried it.